dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
+    <template #title>New Dataset - GDV</template>
+    <template #content>
+      <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+        <p class="font-medium text-xl">Loading GDV data...</p>
+        <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
+      </div>
+      <div v-else class="grid uploadFormWrapper">
+        <div id="uploadForm" class="text-left uploadForm col-9">
+          <FormKit
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/ba067409-ee4b-4972-916f-3b99a63317de)

Can we change the placeholder text to something like "Add a description for this document" and make it non optional?

Reply:
@MarcinStanczyk7

Reply:
Do we want the comment to be required or do we just want to change the placeholder?

Reply:
Should be required, because in the backend it is expected as non nullable

Reply:
This is **not a comment**. It is the `value` field of a BaseDataPointString type object.

Reply:
changed to required

Reply:
ok